### PR TITLE
Updating some broken links 

### PR DESCRIPTION
**Pull request by EziOzoani**

### CODE REVIEW

File: docs/hub/model-card-landscape-analysis.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ We began by analysing language model, model cards, in order to identify patterns
 
 From our analysis of all the models on the hub, we noticed that the most downloads come from top 200 models.
 
-<div class="flex justify-center">
```
Comment (osanseviero) [inline docs/hub/model-card-landscape-analysis.md:102]: This was fine, but https://github.com/huggingface/hub-docs/pull/921 is needed as well

This helps centers the image

File: docs/hub/model-card-landscape-analysis.md [hunk 2]
```diff
@@ -24,12 +24,12 @@ These groupings are not mutually exclusive; they do include overlapping aspects
...
+| DATA                             	| ***CrowdWorkSheets***  [(Díaz et al., 2022)](https://arxiv.org/pdf/2206.08931.pdf)                                                         	| “We introduce a novel framework, CrowdWorkSheets, for dataset developers to facilitate transparent documentation of key decisions points at various stages of the data annotation pipeline: task formulation, selection of annotators, plat- form and infrastructure choices, dataset analysis and evaluation, and dataset release and maintenance.”                                                 	| See [(Díaz et al., 2022)](https://arxiv.org/pdf/2206.08931.pdf)                                                                                          	|
```
Comment (osanseviero) [inline docs/hub/model-card-landscape-analysis.md:27]: The arxiv links could also be hf.co/papers links

**Review by osanseviero — COMMENTED** [review]
Thank you!

File: docs/hub/model-card-landscape-analysis.md (hunk 2 above)
Comment (EziOzoani) [reply docs/hub/model-card-landscape-analysis.md:27]: I literally thought that was I pushing 
I'll update them now

File: docs/hub/model-card-landscape-analysis.md (hunk 1 above)
Comment (EziOzoani) [reply docs/hub/model-card-landscape-analysis.md:102]: ahh a wandering div
fixed
thank you

**Review by osanseviero — APPROVED** [review]
LGTM

**Comment by julien-c** [comment]
this can be merged no @EziOzoani?

**Pull request merged**